mender/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## master
 * Fix mongodb uri creation when using the mongodb subchart and replicast architecture
+* Replaced minio specific gateway with a generic `storage_gateway` service, that could
```

Comment:
This is a breaking change -- should deserve a major version bump in the `Chart.yaml`

Reply:
Right! Why not to use a drop-in replacement?

Keep using both the keys:
```
api_gateway:
  minio:
    enabled: true
```
and:
```
api_gateway:
  storage_proxy:
    enabled: false
```